jaxb/src/main/java/feign/jaxb/JAXBDecoder.java

```diff
@@ -69,6 +70,10 @@ public Object decode(Response response, Type type) throws IOException {
       return Util.emptyValueOf(type);
     if (response.body() == null)
       return null;
+    while (type instanceof ParameterizedType) {
+      ParameterizedType ptype = (ParameterizedType) type;
```

Comment:
Would be nice to have a test covering this loop

Reply:
test added

Reply:
Sweet, thanks! I will review it right now